presenters/table_presenter.go

```diff
@@ -161,6 +161,79 @@ func (t TablePresenter) PresentPendingChanges(output api.PendingChangesOutput) {
 	t.tableWriter.Render()
 }
 
+func (t TablePresenter) PresentProducts(products models.ProductsVersionsDisplay) {
+	var columns []string
+	if products.Available {
+		columns = append(columns, "Available")
+	}
+
+	if products.Staged {
+		columns = append(columns, "Staged")
+	}
+
+	if products.Deployed {
+		columns = append(columns, "Deployed")
+	}
+
+	t.tableWriter.SetAlignment(tablewriter.ALIGN_LEFT)
+	t.tableWriter.SetHeader(append([]string{"Name"}, columns...))
+
+	for _, product := range products.ProductVersions {
+		moreAvailableProducts, firstProductLine := true, true
+		hasData := false
+
+		for moreAvailableProducts {
+			var outputData []string
+
+			moreAvailableProducts = false
+			if products.Available {
```

Comment:
lost of nested `if`s in these `for` loops.
This is still clear, but aesthetically could put the logic of these if blocks into functions to make the nesting a little less crazy

Reply:
Makes sense. I was definitely running short on time when doing this and wanted something working before the refactoring step.